src/util/files.ts

```diff
@@ -127,7 +127,7 @@ export async function readLineByLine(filePath: string, onLine: (line: Buffer, li
  * See {@link readLineByLine} for an asynchronous version.
  */
 export function readLineByLineSync(filePath: string, onLine: (line: Buffer, lineNumber: number) => void, maxLines: number = Infinity): boolean {
-	if(!fs.existsSync(filePath)) {
+	if(!fs.existsSync(filePath) || !fs.statSync(filePath).isFile()) {
```

Comment:
`statSync` should include `existsSync` (it is just slightly more expensive) so we can drop the first part!
```suggestion
	if(!fs.statSync(filePath).isFile()) {
```

Reply:
I would prefer the try catch to avoid race conditions if the file vanishes